lua/telescope/builtin.lua

```diff
@@ -236,6 +236,20 @@ builtin.grep_string = function(opts)
   }
 end
 
+builtin.oldfiles = function()
+  local oldfiles_finder = finders.new {
+    results = vim.tbl_map(
+      function(x) return (x:gsub('\n', '')) end,
```

Comment:
Could also use `require('plenary.functional').first(x:gsub('\n', ''))` here, which (from the tests I did when working on https://github.com/nvim-lua/plenary.nvim/pull/10) is just a bit faster.

Reply:
You can just use `vim.split` maybe?

Reply:
How so? I got an error there regarding newlines being in the vim.v.oldfiles string, so this line is to remove those newlines; is that not necessary/is there a better way?

Reply:
I was thinking `results = vim.split(vim.v.oldfiles)` or something.

Reply:
Hmm, that won't work since vim.v.oldfiles is a table, but I am not having the issue with there being a newline in the vim.v.oldfiles strings, so I just switched it to `results = vim.v.oldfiles`.